rust/core-lib/src/editor.rs

```diff
@@ -169,7 +169,8 @@ impl<W: Write + Send + 'static> Editor<W> {
         self.views.len() > 0
     }
 
-    pub fn set_path<P: AsRef<Path>>(&mut self, path: P) {
+    /// should only ever be called from `BufferContainerRef::set_path`
+    pub fn _set_path<P: AsRef<Path>>(&mut self, path: P) {
```

Comment:
Another pattern to signal the same is `#[doc(hidden)]`, especially given that you haven't written actual doc. But the underscore is ok here.